hypothesis-python/tests/cover/test_datetimes.py

```diff
@@ -135,3 +137,15 @@ def test_can_generate_naive_time():
 @given(times())
 def test_naive_times_are_naive(dt):
     assert dt.tzinfo is None
+
+
+# The fold attribute was added in Python 3.6, but is also missing from PyPy.
+# See https://bitbucket.org/pypy/pypy/issues/2987 for details.
```

Comment:
The `fold` attribute is not missing from PyPy 3.6:

```python
>>>> from datetime import datetime
>>>> datetime(2020, 1, 1, fold=1)
datetime.datetime(2020, 1, 1, 0, 0, fold=1)
```

The referenced bug relates to PyPy's shims for the datetime C API - the datetime C API added functions related to handling fold, and PyPy didn't update their shims, so those are not accessible to C extensions, but the structure itself does support PEP 495.

Reply:
Hmm.  I'll amend the comments, but these tests were definitely failing on `pypy3.6`.  Maybe it's automatically canonicalising `fold` to `0` unless it's actually an ambiguous time?  Regardless I'm going to leave it skipped, as I care more about testing the logic than implementation-specific details.